add-on/preferences/prefs.js

```diff
@@ -1,7 +1,12 @@
 pref("loop.enabled", true);
 pref("loop.remote.autostart", true);
+#ifdef LOOP_DEV_XPI
```

Comment:
I wanted to use LOOP_DEV_XPI so that when it gets into mozilla-central, its clear its a variable belonging to us.